dali/gear/general.py

```diff
@@ -279,6 +279,26 @@ class GoToLastActiveLevel(_StandardCommand):
     _cmdval = 0x0a
 
 
+class ContinuousUp(_StandardCommand):
+    """Dim up using the set fade rate.
+
+    targetLevel shall be set to maxLevel and a fade shall be started
+    using the set fade rate. The fade shall stop when maxLevel is
+    reached.
+    """
+    _cmdval = 0x0b
+
+
+class ContinuousDown(_StandardCommand):
+    """Dim down using the set fade rate.
+
+    targetLevel shall be set to minLevel and a fade shall be started
+    using the set fade rate. The fade shall stop when minLevel is
+    reached.
+    """
+    _cmdval = 0x0c
+
+
```

Comment:
I can't find ContinuousUp and ContinuousDown in my copy of the standard; opcodes 0x0b and 0x0c aren't listed.  Do you have a reference for these?

Changes to the supported commands would need to be in a separate pull request.

Reply:
I have the most resent DALI standard where the new commands are listed. I can send you a screenshot by email.

Reply:
That would be great, thank you.